src/test/java/org/jitsi/meet/test/AudioOnlyTest.java

```diff
@@ -59,24 +59,10 @@ public void enableAudioOnlyAndCheck()
             true);
     }
 
-    /**
-     * Verifies that participant1 cannot video unmute while in audio only mode.
-     */
-    @Test(dependsOnMethods = { "enableAudioOnlyAndCheck" })
-    public void videoUnmuteDisabledInAudioOnly()
-    {
-        getParticipant1().getToolbar().clickAudioMuteButton();
```

Comment:
The impl doesn't match the description. Also, user is allowed to unmute while in audio only mode.